include/Graph/Graph.hpp

```diff
@@ -355,6 +356,19 @@ namespace CXXGRAPH
 		* @param start Node from where traversing starts
 		* @return a vector of nodes that belong to C but not to M.
 		*/
+
+		virtual std::vector<std::vector<Node<T>>> kosaraju() const;
+		/**
```

Comment:
Please introduce the Doxygen documentation before the function declaration, and pay attention to not mess up the documentation of other method/function

Reply:
Made the requested changes.
Corrected the commenting.
Moved the Doxygen documentation before the function declaration and corrected the documentation of the other function that I messed up earlier.